App/Course App/Course App/SharedUI/Styles/SelectableButtonStyle.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SelectableButtonStyle.swift
+//  Course App
+//
+//  Created by Christián on 19/05/2024.
+//
+
+import SwiftUI
+
+struct SelectableButtonStyle: ButtonStyle {
+    var color: Color
+    @Binding var isSelected: Bool
+    
+    enum MagicNumbers: Double {
```

Comment:
I would suggest better naming, eg Constants (it's very generic but the enum could be private to be available just here). On the other side I like the idea with Double enum and cases names :-D